OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -25,6 +25,7 @@ namespace OpenRA.Mods.Common.Widgets.Logic
 {
 	public class LobbyLogic : ChromeLogic
 	{
+		public const string OpenRALANGame = "OpenRALANGame";
```

Comment:
It is advertised after host creates game so also before the first player joins and also after each change in map or players/bots. Re 'Started' games I didn't consider it a real issue as we are mainly interested in Waiting games for LAN. Advantage of LobbyLogic is, that it is all in one place with all information (start, change of map, change of players,.). I will move it to MasterServerPinger. Do you want to keep advertising also Started games?